### DISCUSSION TITLE
I need ESC to stay off for longer

### DISCUSSION THREAD

**Discussion by DavidIgor** [cedricp/ddt4all#998] [General]
esc button will turn off traction control and stability control but it will activate it back after car reaches around 30Km/h . is there way to increase that threshold or even turn it off? 
Dacia Dokker 2018 h4m

**Comment by AndyIreland** [comment]
I dont know on the Dokker - it must depend on what braking/ ABS ECU that DDT4all detects when scanning with the magnifying glass. 

But yes I have disabled ESC (electronic stabilise control) for testing purposes before on my wifes Clio Mk4 2016 and my ZOE EV 2013 - you will get a light showing up on dashboard when its disabled (and in the case of the clio 'check ESC / check braking" )  but even that can normally be extinguished with DDT through the Cluster/TDB ECU 

******Do at your own risk! ******** - especially changing parameters on Braking and safety like that!

**Reply by DavidIgor** [reply]
@AndyIreland yeah, not disabling it permanently... just increasing the speed when it kicks back on

**Reply by AndyIreland** [reply]
I have never seen that setting on our ECU's for the Clio and Zoe - just parameters for disabling and enabling ESC/ASR
